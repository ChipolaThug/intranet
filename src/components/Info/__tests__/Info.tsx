/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react"
import { MemoryRouter } from "react-router-dom"

import Info from "../index"

describe("<Info />", () => {
    it("renders", () => {
        const tree = render(
            <MemoryRouter>
                <Info
                    item={{
                        id: 1,
                        name: "PeL",
                        phone: "+886 0970...",
                        email: "forceoranj@gmail.com",
                        website: "https://www.parisestludique.fr",
                    }}
                />
            </MemoryRouter>
        ).container.firstChild

        expect(tree).toMatchSnapshot()
    })
})
