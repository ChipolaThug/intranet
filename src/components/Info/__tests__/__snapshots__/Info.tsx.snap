// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Info /> renders 1`] = `
<div
  class="UserCard"
>
  <h4>
    User Info
  </h4>
  <ul>
    <li>
      Name: 
      PeL
    </li>
    <li>
      Phone: 
      +886 0970...
    </li>
    <li>
      Email: 
      forceoranj@gmail.com
    </li>
    <li>
      Website: 
      https://www.parisestludique.fr
    </li>
  </ul>
</div>
`;
